src/nvim/event/rstream.c

```diff
@@ -89,7 +89,10 @@ static void on_rbuffer_nonfull(RBuffer *buf, void *data)
 static void alloc_cb(uv_handle_t *handle, size_t suggested, uv_buf_t *buf)
 {
   Stream *stream = handle->data;
-  buf->base = rbuffer_write_ptr(stream->buffer, &buf->len);
+  // `uv_buf_t.len` happens to have different size on Windows.
+  size_t write_count;
+  buf->base = rbuffer_write_ptr(stream->buffer, &write_count);
+  buf->len = write_count;
```

Comment:
A related issue was discussed at https://github.com/libuv/libuv/issues/1233

> In addition, it's not intuitive to have `uv_buf_t`'s `len` property be a `size_t` but the argument to `uv_buf_init()` be an `unsigned int`.

Should be fixed in libuv 2.0.